src/components/SidebarDesktop.jsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import { NavLink as RouterLink } from 'react-router-dom'
+
+import { useInstanceInfo } from 'cozy-client'
+import { makeDiskInfos } from 'cozy-client/dist/models/instance'
+import { isFlagshipApp } from 'cozy-device-helper'
+import flag from 'cozy-flags'
+import CloudIcon from 'cozy-ui/transpiled/react/Icons/Cloud'
+import ContractIcon from 'cozy-ui/transpiled/react/Icons/Contract'
+import DevicesIcon from 'cozy-ui/transpiled/react/Icons/Devices'
+import EmailIcon from 'cozy-ui/transpiled/react/Icons/Email'
+import GlobeIcon from 'cozy-ui/transpiled/react/Icons/Globe'
+import GraphCircle from 'cozy-ui/transpiled/react/Icons/GraphCircle'
+import HandIcon from 'cozy-ui/transpiled/react/Icons/Hand'
+import HelpOutlinedIcon from 'cozy-ui/transpiled/react/Icons/HelpOutlined'
+import JusticeIcon from 'cozy-ui/transpiled/react/Icons/Justice'
+import LockScreen from 'cozy-ui/transpiled/react/Icons/LockScreen'
+import Logout from 'cozy-ui/transpiled/react/Icons/Logout'
+import OpenwithIcon from 'cozy-ui/transpiled/react/Icons/Openwith'
+import PaletteIcon from 'cozy-ui/transpiled/react/Icons/Palette'
+import PeopleIcon from 'cozy-ui/transpiled/react/Icons/People'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Nav, {
+  NavItem,
+  NavIcon,
+  NavText,
+  genNavLinkForV6
+} from 'cozy-ui/transpiled/react/Nav'
+import Sidebar from 'cozy-ui/transpiled/react/Sidebar'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+import { usePremium } from '@/components/Premium/PremiumProvider'
+import { routes } from '@/constants/routes'
+import { useLogout } from '@/hooks/useLogout'
+
+export const SidebarDesktop = () => {
+  const { t } = useI18n()
+  const logout = useLogout()
+  const { isLoaded, instance, diskUsage } = useInstanceInfo()
+  const { canOpenPremiumLink, premiumLink } = usePremium()
+
+  const NavLink = genNavLinkForV6(RouterLink)
+  const percent = isLoaded
+    ? makeDiskInfos(diskUsage.data.used, diskUsage.data.quota).percentUsage
+    : ''
+
+  return (
+    <Sidebar style={{ width: 360 }}>
```

Comment:
Can we do direct styling like this?

Reply:
idealy we need to modify Sidebar in cozy-ui to be able to pass classes, but there's no need to do that except in Settings, so this is a quick answer